components/shared/workCard/WorkCard.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from "next/image"
+import Link from "next/link"
+import React from "react"
+
+import { WorkTeaserFragment } from "@/lib/graphql/generated/fbi/graphql"
+
+import { displayCreators } from "./helper"
+import WorkCardAvailabilityRow from "./WorkCardAvailabilityRow"
+
+type WorkCardProps = {
+  work: WorkTeaserFragment
+}
+
+const WorkCard = ({ work }: WorkCardProps) => {
+  return (
+    <div className="mb-4">
+      <Link href={`/work/${work.workId}`}>
+        <div key={work.workId} className="rounded-sm bg-background-overlay p-2 md:p-4">
+          <div
+            className="px-auto shadow-coverPicture relative mx-auto mb-3 mt-6 flex aspect-[166/228] w-[calc(100%-76px-4px)]
+              items-center rounded-sm md:mb-6 md:mt-9 md:w-[calc(100%-106px)]">
+            <Image
+              src="https://i.pinimg.com/564x/b0/4a/b2/b04ab2de27591aee4a76015e42e0e282.jpg"
```

Comment:
If we use demo content like this we need to write a `TODO`about that real values should be used in the future.